client/components/forms/form-text-input/test/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import chai, { expect } from 'chai';
+import { shallow } from 'enzyme';
+import { spy } from 'sinon';
+import chaienzyme from 'chai-enzyme';
+import FormTextInput from 'components/forms/form-text-input';
+
+chai.use( chaienzyme() );
```

Comment:
You don't need this line, since we [add all of the Chai plugins during the test setup](https://github.com/Automattic/wp-calypso/blob/e6eb37a38c8f5041e3cdf285bea03afc8ef989fc/test/boot-test.js#L12). Make sure to also remove the `chai` and `chaienzyme` imports.